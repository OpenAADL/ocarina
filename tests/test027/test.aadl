-- test for parsing Import declaration in package

package Satellite
  public
    with Security::Verification::Position ;
    with Control::Target;
    with Command::Launcher;
    

    Pos   renames package Security::Verification::Position ;
    Proc1 renames process Control::Target::Proc.Impl ;
    renames feature group thefeature ;
    renames Command::Launcher::all ;
    renames Foo::all;

    data position
    end position;

    data commands
    end commands;

  private
    with Error::Abort ;

    data value
    end value;
end Satellite;
