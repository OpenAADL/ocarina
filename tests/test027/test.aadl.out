test.aadl:5:05:  (package name) is not a package or a property set visible or existing
Cannot analyze AADL specifications
